pkg/cmd/pipeline/translate.go

```diff
@@ -260,11 +261,27 @@ func AddDevAnnotations(ctx context.Context, manifest *model.Manifest, c kubernet
 			continue
 		}
 		if repo != "" {
-			app.ObjectMeta().Annotations[model.OktetoRepositoryAnnotation] = repo
+			app.ObjectMeta().Annotations[model.OktetoRepositoryAnnotation] = removeSensitiveDataFromGitURL(repo)
 		}
 		app.ObjectMeta().Annotations[model.OktetoDevNameAnnotation] = devName
 		if err := app.PatchAnnotations(ctx, c); err != nil {
 			oktetoLog.Infof("could not add %s dev annotations due to: %s", devName, err.Error())
 		}
 	}
 }
+
+func removeSensitiveDataFromGitURL(gitURL string) string {
+	if gitURL == "" {
+		return gitURL
+	}
+
+	parsedRepo, err := giturls.Parse(gitURL)
+	if err != nil {
+		return ""
```

Comment:
I couldn't see any line that can be based on this value so I guess is safe to set it as `""` for that cases.

Reply:
i found this https://github.com/okteto/app/master/backend/pkg/model/redact.go#L13 at the backend, where logic is more or less the sale, in case of err, the gitURL is returned, could it be that the redacted function here is not working?

Reply:
Seems this annotations in set from backend installer so this fix need to be done from CLI side. I tested my solution with my change in a custom installer with proposed changes and works as expected 👍

Reply:
@AdrianPedriza does that mean that this PR is not needed?

Reply:
no no, means that change needs to be done in CLI not it backend